src/App/Component/Console/PackageCommand.php

```diff
@@ -346,4 +347,23 @@ protected function getExampleCommandPrefix(): string
         $isBash = ($shell && stripos($shell, 'bash')) !== false;
         return $isBash ? './' : '';
     }
+
+    protected function checkSSHConnection(): void
+    {
+        $process = new Process(['ssh', '-T', 'git@github.com']);
+        $process
+            ->setTimeout(null)
+            ->run();
+
+        if ($process->getExitCode() !== 1) {
+            $this
+                ->getIO()
+                ->error([
+                    'Checking access to github.com ... DENIED',
```

Comment:
Can't we take the error from the `ssh` itself?

Reply:
display a more informative message

As mentioned in the issue, the message is confusing

Reply:
I mean it could be another reason, not only denied access.

Reply:
added errors.

Is it correct to end the command with the exit construct?